src/core/evaluator.js

```diff
@@ -1732,6 +1732,7 @@ class PartialEvaluator {
     const stateManager = new StateManager(initialState);
     const preprocessor = new EvaluatorPreprocessor(stream, xref, stateManager);
     const timeSlotManager = new TimeSlotManager();
+    let markedContentLevel = 0;
```

Comment:
This behaves subtly different than the code in `getTextContent`, which keeps the existing level-state when parsing /Form XObjects.

Perhaps that's not actually necessary to do, but my point here is that `getOperatorList` and `getTextContent` should handle markedContent-levels in /Form XObjects the *same way* (whatever that way is).